packages/cypress/integration/demo_reflect.ts

```diff
@@ -0,0 +1,89 @@
+function addCard(column, text) {
+  cy.get(`[data-cy=phase-item-editor-${column}-wrapper]`)
+      .as('column-phase-item-editor')
+    
+  cy.get('@column-phase-item-editor')
+    .type(`${text}`)
+    .should('have.text', `${text}`)
+
+  cy.get('@column-phase-item-editor')
+    .type('{enter}')
+  
+  cy.wait(500)
+
+}
+
+function editCard(column, oldText, newText) {
+  cy.get(`[data-cy=reflection-stack-${column}-card-0-wrapper]`)
+    .as('reflection-card-edit')
+
+  cy.get('@reflection-card-edit')
+    .should('have.text', `${oldText}`)
+
+  cy.get('@reflection-card-edit')
+    .type('{selectall}').type('{backspace}').type(`${newText}`)
+    .should('have.text', `${newText}`)
+
+}
+
+function deleteCard(column) {
+      
+  cy.get(`[data-cy=reflection-stack-${column}-card-0-delete]`)
+    .as('reflection-card-delete')
+
+  cy.get('@reflection-card-delete').click()
+}
+
+describe('Test Reflect page Demo', () => {
+
+  before(function() {
+    // runs before each test in the block
+    cy.visit('/retrospective-demo/reflect')
+    cy.get('[data-cy=start-demo-button]').should('be.visible').click()
+
+    cy.get('[data-cy=sidebar-header]').find('button').should('be.visible').click()
+
+  })
+
+  it('Test help menu toggle', () => {
+
+    cy.get('[data-cy=help-menu-toggle]').should('be.visible')
+    
+    cy.get('[data-cy=help-menu-close]').should('be.visible').click()
```

Comment:
can remove the extra `should` methods on clicks?

Reply:
The menu close button is a different component than the toggle so I thought I would check if it was visible as well before testing the closing functionality.

Reply:
ah gotcha, fair enough!